module3/Projects/Mod3Project.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: page
+title: Mod3 Project
+---
+
+# Message Logger - The Final Iteration!
+
+## Learning Goals
+- Demonstrate:
+    * Version Control, including branches and pull requests.
+    * MVC code organization within .NET
+    * Feature testing of a .NET application
+
+- Explore:
+    * Usability and creating a quality user experience
+    * Planning out the implementation of a feature
+    * Using TDD to add additional features! (For any new feature that you create, try writing the tests first, and then creating the code that makes those tests pass!)
+
+## Project Description
+For this project, you will build the final iteration of the Message Logger we created in Mod1 and built on in Mod2. In Mod2 you added a database, and now you're going to add a frontend! You are encouraged to bring in code you wrote in the Mod2 project, especially for calculating message statistics.
+
+
+This project will give you the opportunity to showcase all of the concepts we have learned this Mod.  There might be skills that we covered all the way back in week 1, so don't forget to use those earlier lessons and labs as a resource!
+
+**Project Setup**
+To get started on this project:
+
+1. **fork and clone** this [starter repo](https://github.com/turingschool-examples/MvcMessageLogger)
+    * The starter repo includes classes similar to the ones you wrote in your Mod2 project. The context and database connection are set up for you, connecting to a new database called `MvcMessageLogger`.
+2. The test project has not been setup because the expectation is that you will entirely change the UI.
+3. Use the existing migration to create your database and run the application to make sure the project is working.
+
+When you have completed the Project Setup, send a link to your cloned repository to all your instructors.
+
+### Iteration 1 - Creating and Viewing Users
+
+This iteration includes the following features:
+1. A way to see all users
+1. A way to add a new user
+
+A feature is not considered complete until it is tested, so make sure to test both of these features!
+
+Before writing code, there are two planning steps you need to complete and include here in the README.
+1. Draw a request/response diagram that includes all requests required to complete this iteration
+1. Draw a rough wireframe of the UI page(s) required to complete this iteration
+
+Add screenshots of your planning images in your project README, where it says:
+#### REPLACE THIS HEADER WITH YOUR ITERATION 1 REQUEST/RESPONSE IMAGE
+#### REPLACE THIS HEADER WITH YOUR ITERATION 1 WIREFRAME

```

Comment:
I think in the project kickoff we can introduce what wire-framing is - I don't think it is too complex of a topic.  (this is how the BE team introduces wireframing)